README.md

```diff
@@ -559,6 +560,36 @@ The model can also be downloaded and served locally.
 
    The output from the command will consist of a series of outputs from the model before and after training.
 
+### 🧪 Evaulate the newly trained model
+
+`ilab` ships with several benchmarks you can use to evaluate the models you are training. Currently, four benchmarks are supported.
+
+| Benchmark | Measures | Full Name | Description |
+| --- | --- | --- | --- |
+| MMLU | Knowledge | Massive Multitask Language Understanding | TODO | 
+| MTBench | Skills | Multi-turn Benchmark | TODO |
+| MMLUBranch | Knowledge | N/A | TODO |
+| MTBenchBranch | Skills | N/A | TODO |
+
+> [!NOTE]
+> MTBench and MTBenchBranch use [prometheus-8x7b-v2.0](https://huggingface.co/prometheus-eval/prometheus-8x7b-v2.0) by default as a judge model - while you do not __need__ to use this model as your judge, it is strongly recommend. You can download it via `ilab model download`.
+
+#### Running MMLU
+
+1. `ilab model evaluate --benchmark mmlu`
```

Comment:
I usually prefer having single step procedures as bullet points, so it would just be
```suggestion
* `ilab model evaluate --benchmark mmlu`
```
And like that for all four benchmarks

Reply:
This is gonna change pretty significantly as I work through it, but will keep this in mind

Reply:
Gotcha, sounds good!